src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue

```diff
@@ -158,7 +166,7 @@ function removeEndpoint(endpointName, instance) {
       console.log(response);
       endpoint.value.instances.splice(endpoint.value.instances.indexOf(instance), 1);
       if (endpoint.value.instances.length === 0) {
-        router.push({ name: "monitoring", query: { historyPeriod: selectedHistoryPeriod } });
+          router.push({ name: "monitoring", query: { historyPeriod: historyPeriod.value.value } });
```

Comment:
Not necessarily important for this PR, but we may want to check why there seems to be a ref() of a ref() going on here because a value.value is needed.

Reply:
Thats becos the period object has the following properties
 { value: 1, text: "1m", refreshInterval: 1 * 1000, refreshIntervalText: "Show data from the last minute. Refreshes every 1 second" },
 So, when I want to access the value 1, I need to get the value ou tof ref and then the element . I am planning to do another PR to change that name to something like PeriodVal